vimrc

```diff
@@ -36,6 +36,7 @@ set ignorecase
 set smartcase
 set wildignore+=*.pyc,*.o,*.class,*.lo,.git,vendor/*,node_modules/**,bower_components/**,*/build_gradle/*,*/build_intellij/*,*/build/*,*/cassandra_data/*
 set tags+=gems.tags
+set backupcopy=yes " Setting backup copy preserves file inodes, which are needed for Docker file mounting
```

Comment:
`:h 'backupcopy'` says that the default value is `yes` for Unix systems. Weird that you have to explicitly set it?